aya/src/programs/tc.rs

```diff
@@ -228,6 +228,64 @@ define_link_wrapper!(
     TcLinkId
 );
 
+impl SchedClassifierLink {
+    /// This API is used to construct a [`SchedClassifierLink`] where the `if_name`, `attach_type`,
+    /// `priority` and `handle` are already known. This may have been found from a link created by
+    /// [SchedClassifier::attach], the output of the `tc filter` command or from the output of
+    /// another BPF loader.
+    ///
+    /// Note: If you create a link for a program that you do not own, detaching it may have
+    /// unintended consequences.
+    ///
+    /// # Errors
+    /// - If a program is not attached with the provided parameters, calls to
```

Comment:
I would make this (without making it a bullet item):


Returns [`io::Error`] if `if_name` is invalid. If the other parameters are invalid this call
will succeed, but calling [`SchedClassiferLink::detach`] will return [`TcError::NetlinkError`].

Reply:
Great suggestion.  Done.